Blish HUD/GameServices/Modules/Managers/ContentsManager.cs

```diff
@@ -93,11 +94,25 @@ public class ContentsManager : IDisposable {
         }
 
         /// <summary>
-        /// [NOT IMPLEMENTED] Loads a <see cref="BitmapFont"/> from a file.
+        /// Loads a <see cref="BitmapFont"/> from a TrueTypeFont (*.ttf) file.
         /// </summary>
-        /// <param name="fontPath">The path to the font file.</param>
-        public BitmapFont GetBitmapFont(string fontPath) {
-            throw new NotImplementedException();
+        /// <param name="fontPath">The path to the TTF font file.</param>
+        /// <param name="size">Size of the font.</param>
+        public BitmapFont GetFont(string fontPath, ContentService.FontSize size) {
+            long fontDataLength = _reader.GetFileBytes(fontPath, out byte[] fontData);
+
+            if (fontDataLength > 0) {
+                using var ctx = GameService.Graphics.LendGraphicsDeviceContext();
+                var bakeResult = TtfFontBaker.Bake(fontData, (int)size, 1024, 1024, new[] {
```

Comment:
Do we have any indication of how much can fit on each of these render surfaces?  Is 1024 chosen arbitrarily, and what range of sizes can it hold if we're putting all Latin, Latin1Supplement, Latin extended, and Cyrillic on it?  😅

Reply:
**Tests**

- Maximum FontSize.36
- Removed CharacterRange.Cyrillic
- Fonts: Lato-Regular.ttf, PTSerif-Regular, GWTwoFont, Cagliostro-Regular.ttf, StoweOpenFace

PTSerif-Regular: Working good.
![image](https://github.com/blish-hud/Blish-HUD/assets/13819164/12a5ef89-5dfb-4a51-a359-5cecc32d9ea9)

 GWTwoFont: Working good.
![image](https://github.com/blish-hud/Blish-HUD/assets/13819164/8c026cef-cb69-41bb-9d53-8226b57bd9bc)

Cagliostro-Regular: Working good
![image](https://github.com/blish-hud/Blish-HUD/assets/13819164/40757e4a-dda8-4000-ae9d-cf1ab61f5144)

StoweOpenFace: Line height seems wrong. But that might be an issue with the font itself or since it is a title font the line height is supposed to overlay a bit by design.
![image](https://github.com/blish-hud/Blish-HUD/assets/13819164/b13fb121-c146-4dc5-8bbf-67f4d85b9bf8)

Result: Added optional line height parameters for "wonky" font line heights. I expect none to be wonky like StowOpenFace. It's likely by design. But able to set a custom line height is good anyway.